lib/modules/manager/npm/update/dependency/index.ts

```diff
@@ -174,6 +187,7 @@ export function updateDependency({
         depName,
         oldVersion!,
         `npm:${upgrade.newName}@${newValue}`,
+        isYaml,
```

Comment:
I'd rather forward the packageFile here, which can be used for different purposes here too.